### PR TITLE
#973 clearly document the tiered storage config values

### PR DESCRIPTION
**Pull request by jrkinley**

### CODE REVIEW

**Review by alejandroEsc — APPROVED** [review]
lgtm, thank you!

File: charts/redpanda/values.yaml [hunk 1]
```diff
@@ -470,40 +470,49 @@ storage:
...
+      # -- AWS or GCP access key (required for AWS and GCS authentication with access keys).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_access_key).
       cloud_storage_access_key: ""
-      # -- Required for AWS and GCS authentication with access keys.
+      # -- AWS or GCP secret key (required for AWS and GCS authentication with access keys).
       # See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_secret_key).
       cloud_storage_secret_key: ""
-      # -- See the [property reference documentation](https://docs.redpanda.com/docs/reference/cluster-properties/#cloud_storage_api_endpoint).
+      # -- AWS or GCP API endpoint.
+      # - For AWS, this can be left blank as it is generated automatically using the bucket and region (e.g. "<bucket>.s3.<region>.amazonaws.com")
```
Comment (github-amine-kherbouche) [inline charts/redpanda/values.yaml:492]: for AWS it ***should be*** left blank if the region and bucket name are set

**Review by github-amine-kherbouche — COMMENTED** [review]
my 2 cents

File: charts/redpanda/values.yaml (hunk 1 above)
Comment (RafalKorepta) [reply charts/redpanda/values.yaml:492]: We have the same wording in our docs https://docs.redpanda.com/current/manage/tiered-storage/#:~:text=For%20AWS%2C%20this%20can%20be%20left%20blank.%20It%E2%80%99s%20generated%20automatically%20using%20the%20region%20and%20bucket

Could you file an issue in https://github.com/redpanda-data/docs/issues? 

Then we would change in both places.

**Pull request merged**